From fda00443cc0a848cf52a85367f79cd99dae5073c Mon Sep 17 00:00:00 2001
From: fundamental <mark.d.mccurry@gmail.com>
Date: Thu, 4 Apr 2013 23:12:33 -0400
Subject: [PATCH 0068/1936] Updates install process from AUR hints

- Installs files based upon arch linux PKGBUILD
- Adds desktop files to provide shortcuts and window icon support
Thanks arch linux packager
---
 CMakeLists.txt                              |   6 ++++++
 zynaddsubfx-alsa.desktop                    |   8 ++++++++
 zynaddsubfx-jack.desktop                    |   8 ++++++++
 zynaddsubfx_icon.ico => zynaddsubfx.ico     | Bin
 src/zynaddsubfx_icon.svg => zynaddsubfx.svg |   0
 5 files changed, 22 insertions(+)
 create mode 100644 zynaddsubfx-alsa.desktop
 create mode 100644 zynaddsubfx-jack.desktop
 rename zynaddsubfx_icon.ico => zynaddsubfx.ico (100%)
 rename src/zynaddsubfx_icon.svg => zynaddsubfx.svg (100%)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index edc7d27e..e7963b0b 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -11,3 +11,9 @@ add_subdirectory(src)
 install(FILES AUTHORS.txt COPYING FAQ.txt HISTORY.txt README.txt
 	DESTINATION share/doc/zynaddsubfx
 	)
+install(FILES zynaddsubfx-jack.desktop zynaddsubfx-alsa.desktop
+    DESTINATION share/applications)
+install(FILES zynaddsubfx.svg
+    DESTINATION share/pixmaps)
+install(DIRECTORY instruments/banks
+    DESTINATION share/zynaddsubfx)
diff --git a/zynaddsubfx-alsa.desktop b/zynaddsubfx-alsa.desktop
new file mode 100644
index 00000000..2b76509e
--- /dev/null
+++ b/zynaddsubfx-alsa.desktop
@@ -0,0 +1,8 @@
+[Desktop Entry]
+Name=ZynAddSubFX - Alsa
+Comment=A powerful realtime software synthesizer
+Exec=zynaddsubfx -I alsa -O alsa
+Icon=zynaddsubfx
+Terminal=false
+Type=Application
+Categories=Application;AudioVideo;
diff --git a/zynaddsubfx-jack.desktop b/zynaddsubfx-jack.desktop
new file mode 100644
index 00000000..51305c6e
--- /dev/null
+++ b/zynaddsubfx-jack.desktop
@@ -0,0 +1,8 @@
+[Desktop Entry]
+Name=ZynAddSubFX - Jack
+Comment=A powerful realtime software synthesizer
+Exec=zynaddsubfx -I jack -O jack
+Icon=zynaddsubfx
+Terminal=false
+Type=Application
+Categories=Application;AudioVideo;
diff --git a/zynaddsubfx_icon.ico b/zynaddsubfx.ico
similarity index 100%
rename from zynaddsubfx_icon.ico
rename to zynaddsubfx.ico
diff --git a/src/zynaddsubfx_icon.svg b/zynaddsubfx.svg
similarity index 100%
rename from src/zynaddsubfx_icon.svg
rename to zynaddsubfx.svg
-- 
2.47.0

